scripts/run.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+password_url="taskcluster/secrets/v1/secret/repo:github.com/mozilla/balrog:dockerhub"
+repo_token=$(curl ${password_url} | python -c 'import json, sys; a = json.load(sys.stdin); print a["secret"]["repo_token"]')
```

Comment:
Yeah, it's up to you to do this though. You can probably check for $GITHUB_HEAD_REPO_URL and only run coveralls if that is set.